harcownik-frontend/src/components/raport/Raport.jsx

```diff
@@ -1,30 +1,29 @@
+/* eslint-disable */
 import React from "react";
-import { Row, Col } from "react-bootstrap";
+import { Row, Col, Container } from "react-bootstrap";
+import "./raport_style.css";
```

Comment:
I am following structure where I have firstly imports external packages, secondly own packages and at the end style files. To summarise it can be  updated but it is not necessary